transformation/src/jade-schema/tables/library.table.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "library",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "source",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "award",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "lab",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "lot_id",
+      "datatype": "string"
+    },
+    {
+      "name": "product_id",
+      "datatype": "string"
+    },
+    {
+      "name": "queried_rnp_size_range",
+      "datatype": "string"
+    },
+    {
+      "name": "rna_integrity_number",
+      "datatype": "float"
+    },
+    {
+      "name": "size_range",
```

Comment:
What do the values for this look like in ENCODE? Could we instead store int `_min` and `_max` columns?

Reply:
It sounds like this is not in a consistent format at the moment (they store this as a single string).

Reply:
And Kathy would add that this is unlikely to be a search criteria.